features.md

```diff
@@ -5,6 +5,18 @@
    <!-- Explain how data is stored and secured on the device and the SSL connection with the server -->
    
 * ### Login
+   1. Entering the server URL as  https://demo.openmrs.org/openmrs 
+   ![OpenMRS Android Client Offline login screen](https://user-images.githubusercontent.com/24738188/34084262-1732ca6c-e3a3-11e7-9e05-96d1ce0835c3.png) 
```

Comment:
Please change the URL shown in the screenshot to `https://demo.openmrs.org/openmrs` otherwise, it will create confusion.